src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -101,24 +101,24 @@ public static Email parseEmail(String email) throws ParseException {
      *
      * @throws ParseException if the given {@code tag} is invalid.
      */
-    public static Tag parseTag(String tag) throws ParseException {
+    public static Tier parseTag(String tag) throws ParseException {
         requireNonNull(tag);
         String trimmedTag = tag.trim();
-        if (!Tag.isValidTagName(trimmedTag)) {
-            throw new ParseException(Tag.MESSAGE_CONSTRAINTS);
+        if (!Tier.isValidTagName(trimmedTag)) {
+            throw new ParseException(Tier.MESSAGE_CONSTRAINTS);
         }
-        return new Tag(trimmedTag);
+        return new Tier(trimmedTag);
     }
 
     /**
      * Parses {@code Collection<String> tags} into a {@code Set<Tag>}.
      */
-    public static Set<Tag> parseTags(Collection<String> tags) throws ParseException {
+    public static Set<Tier> parseTags(Collection<String> tags) throws ParseException {
         requireNonNull(tags);
-        final Set<Tag> tagSet = new HashSet<>();
+        final Set<Tier> tierSet = new HashSet<>();
         for (String tagName : tags) {
-            tagSet.add(parseTag(tagName));
+            tierSet.add(parseTag(tagName));
         }
-        return tagSet;
+        return tierSet;
```

Comment:
Perhaps we could remove this, as tiers are now a singular value and no longer a set.